app/scripts/services/validateRegistrant.js

```diff
@@ -173,7 +173,7 @@ angular
       );
     };
 
-    var blockInRegistrantType = function(block, registrant) {
+    this.blockInRegistrantType = function(block, registrant) {
```

Comment:
How come you made this an instance method? I think it's fine either way but usually you expose it on the class if  you need it in the template or want to share it externally. Just wondering the motivation.

Reply:
I think I did this when I was testing a different solution that required exposing blockInRegistrantType, but it did not end up working. I can change it back because it isn't necessary any more.

Reply:
This could be a `const`.